advertise.md

```diff
@@ -0,0 +1,86 @@
+limitations under the License.
+-->
+{% include JB/setup %}
+<div class="advertise">
+  <div class="advertise container">
+    <h2>Technologies</h2>
+    <div class="border row">
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top: 20px;">
+            <img src="./assets/themes/zeppelin/img/spark_logo.jpg" width="140px">
+          </label>
+          <label class="_hover-text">
+            Spark interpreter group which consists of five interpreters which are Spark, Pyspark, SparkR, Spark SQL, Dependency loader.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/spark_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>                     
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="font-size:56px;">SQL</label>
+          <label class="_hover-text">
+            JDBC interpreter lets you create a JDBC connection to any data sources seamlessly. Postgresql, Mysql, MariaDB, Redshift, Apache Hive and so on.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/sql_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top:65px;">
+            <img src="./assets/themes/zeppelin/img/python-logo-generic.svg" width="200px">
+          </label>
+          <label class="_hover-text">
+            Python is supported with Zeppelin. Also integrated Matplotlib with Python & Pyspark interpreter, and Conda is now available in Zeppelin.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/python_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-12 col-sm-12 col-xs-12 text-center">
+      <p style="text-align:center; margin-top: 20px; font-size: 17px; color: gray; font-weight: 200; font-style: italic; padding-bottom: 0;">
+        See more details in Zeppelin supports 20+ different interpreters.
+        <a href="/docs/0.8.0-SNAPSHOT/manual/interpreters.html">LEARN MORE <span class="glyphicon glyphicon-chevron-right" style="font-size:15px;"></span></a>
+      </p>
+    </div>    
+    <hr />
+    <div class="border row">
+      <h2>Deployments</h2>
+      <div class="border col-md-6 col-sm-6">
+        <p class="panel-content-user">
+          <label style="width: 100%;">
+            <span class="glyphicon glyphicon-user" style="font-size:30px;position:relative;width:100%;text-align:center;"></span>
```

Comment:
MINOR: Can we use http://fontawesome.io/icon/user/ instead of `glyphicon-user`? 

I know they have the same look. But it would be nice to use the same font classes with `fa-users` for consistency.